calico-enterprise/networking/configuring/node-local-dns-cache.mdx

```diff
@@ -0,0 +1,103 @@
+---
+description: Improve cluster DNS performance with NodeLocal DNSCache
+---
+
+# Improve cluster DNS performance with NodeLocal DNSCache
+
+## Big picture
+
+NodeLocal DNSCache deploys `node-local-dns` pods as hostnetwork on all cluster nodes to improve DNS perfomance.
```

Comment:
I'd like to avoid any phrasing that implies we're supporting nodelocaldnscache itself: we aren't - if it has a horrible bug then we don't want to hear about it.

Maybe "If you choose to use NodeLocal DNSCache [link] in your cluster, follow these steps to enable connectivity for Tigera components."